Author: Yongzhi Pan <panyongzhi@gmail.com>
Description: Change locale path for deb package.
Last-Update: 2013-11-23

--- a/4digits
+++ b/4digits
@@ -40,8 +40,8 @@
 import pango
 
 # For distribution packaging
-#LOCALE_PATH = '/usr/share/locale/'
-LOCALE_PATH = 'locale/'
+LOCALE_PATH = '/usr/share/locale/'
+#LOCALE_PATH = 'locale/'
 
 try:
     import pygtk
--- a/4digits-text.c
+++ b/4digits-text.c
@@ -44,8 +44,8 @@
 #define N_(str) str
 
 // For distribution packaging
-//#define LOCALE_PATH "/usr/share/locale/"
-#define LOCALE_PATH "locale/"
+#define LOCALE_PATH "/usr/share/locale/"
+//#define LOCALE_PATH "locale/"
 
 //#define DEBUG
 #define VERSION_STRING "1.1.4"
